app/build.gradle

```diff
@@ -46,4 +51,20 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.1.5'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
     implementation("androidx.activity:activity-ktx:1.6.1")
+    implementation("androidx.core:core-splashscreen:1.0.0")
+    implementation platform('com.google.firebase:firebase-bom:31.3.0')
+    implementation 'com.google.firebase:firebase-analytics-ktx'
+    implementation 'com.google.firebase:firebase-crashlytics-ktx'
+
+
+
+    def room_version = "2.4.3"
```

Comment:
Are you sure this is the version that we need? is this the latest ?

Reply:
Yes it is